jest-e2e.config.js

```diff
@@ -1,21 +1,27 @@
-const { defaults: tsjPreset } = require('ts-jest/presets');
+const { defaults: tsjPreset } = require("ts-jest/presets");
 
 module.exports = {
   ...tsjPreset,
-  preset: 'react-native',
+  preset: "react-native",
   transform: {
-    ...tsjPreset.transform,
-    '\\.js$': 'ts-jest'
+    "^.+\\.ts?$": "ts-jest",
+    "^.+\\.tsx?$": "ts-jest",
+    "^.+\\.js$": "<rootDir>/node_modules/react-native/jest/preprocessor.js"
   },
   transformIgnorePatterns: [
-    "node_modules/(?!react-native|react-navigation|react-native-device-info|native-base|native-base-shoutem-theme|@shoutem/animation|@shoutem/ui|jsbarcode)"
+    "node_modules/(?!(jest-)?@react-native|react-native|react-navigation|@react-navigation|react-navigation-redux-helpers|react-native-device-info|native-base|native-base-shoutem-theme|@shoutem/animation|@shoutem/ui|rn-placeholder|jsbarcode|@pagopa/react-native-cie|react-native-share|jail-monkey|@react-native-community/art|@react-native-community/push-notification-ios|@react-native-community/cameraroll|@codler|@react-native-community/datetimepicker)"
   ],
+  moduleNameMapper: {
+    "\\.svg": "<rootDir>/ts/__mocks__/svgMock.js"
+  },
   globals: {
-    'ts-jest': {
-      babelConfig: true,
+    "ts-jest": {
+      babelConfig: true
     }
   },
-  testMatch: ['**/__e2e__/*.e2e.ts?(x)'],
+  setupFiles: ["./jestSetup.js"],
+  setupFilesAfterEnv: ["@testing-library/jest-native/extend-expect"],
+  testMatch: ["**/__e2e__/*.e2e.ts?(x)"],
   forceExit: true,
```

Comment:
```suggestion
  forceExit: true,
  collectCoverage: true
```
I don't know why github doesn't show (at least at me) the right diff, but is missing `collectCoverage: true` from the original config.

<img width="1734" alt="Schermata 2021-09-09 alle 15 37 00" src="https://user-images.githubusercontent.com/26501317/132696165-5fa1a675-f07e-46f1-a605-e684af6c3e20.png">

Reply:
Nvm, I was comparing the wrong files 😅